notes.txt

```diff
@@ -0,0 +1,37 @@
+pushing docker images to local registry
```

Comment:
notes are great! but they don't belong in the repo 😉 

some of these things might be good in the docs if not already there.